modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -12,6 +12,14 @@ def self.stamp_pdf(stamped_template_path, data)
         stamp_method = "stamp#{data['form_number'].gsub('-', '')}".downcase
         send(stamp_method, stamped_template_path, data)
       end
+
+      # This is a one-off case where we need to stamp a date on the first page of 21-4142 when resubmitting
+      if data['in_progress_form_created_at'] && data['form_number'] == '21-4142'
```

Comment:
does this code need to go here in this generic stamp_pdf?  would it not work in the below stamp214142?

Reply:
Yeah it could go either place. Both have to be executed for resubmitted 4142s. I guess I preferred to put it up top because it'll be more obvious if we want to remove it later on. I don't know of a strong argument going either way on it tbh.